Source/Scene/Camera.js

```diff
@@ -3181,6 +3181,23 @@ Camera.prototype.cancelFlight = function () {
   }
 };
 
+/**
+ * Complete the current camera flight if one is in progress
+ * The camera immediately move to destination
+ */
+Camera.prototype.completeFlight = function () {
+  if (defined(this._currentFlight)) {
+    this.cancelFlight();
+
+    newOptions.duration = 0;
```

Comment:
Do we need to set the `duration` here? It doesn't look like `setView` takes a duration so wouldn't it be ignored either way?